lib/matplotlib/quiver.py

```diff
@@ -626,19 +626,19 @@ def _angles_lengths(self, U, V, eps=1):
 
     def _make_verts(self, U, V, angles):
         uv = (U + V * 1j)
-        str_angles = isinstance(angles, six.string_types)
-        if str_angles and (angles == 'xy' and self.scale_units == 'xy'):
+        str_angles = angles if isinstance(angles, six.string_types) else ''
```

Comment:
yes, but the empty string evaluates to False (as in `print('foo' if '' else 'bar')`), thus short-circuiting the angles == 'xy' test, which can eventually throw a warning.